client/src/app/panel/tabs/linting/LintingTab.js

```diff
@@ -91,7 +91,7 @@ function LintingTabItem(props) {
 
   const {
     category,
-    documentation = {},
+    meta = {},
```

Comment:
```suggestion
    meta,
```

Given the later line:

```
  const documentationUrl = meta?.documentation?.url;  
```

We get away with meta not existing.